x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/helpers.tsx

```diff
@@ -27,7 +28,12 @@ import { BuildQueryBarDescription, BuildThreatDescription, ListItems } from './t
 import { SeverityBadge } from '../severity_badge';
 import ListTreeIcon from './assets/list_tree_icon.svg';
 
-const isNotEmptyArray = (values: string[]) =>
+const NoteDescriptionContainer = styled(EuiFlexItem)`
+  height: 105px;
+  overflow-y: hidden;
+`;
+
+export const isNotEmptyArray = (values: string[]) =>
   !isEmpty(values) && values.filter(val => !isEmpty(val)).length > 0;
```

Comment:
Now that you have tests for this, a simpler implementation would be:

```ts
export const isNotEmptyArray = (values: string[]): boolean => !isEmpty(values.join(''));
```

No changes needed since you didn't write this, just mentioning it.

Reply:
I like that. I think it's easier to read. I went ahead and updated it. Now that there's tests it was easy to change and test  :)